### DISCUSSION TITLE
✅ 🚀 Feature Request: Responsive Mode Toggle: Option to have on by default

### DISCUSSION THREAD

**Discussion by mjagger88** [elementor/elementor#21221] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When building themes and pages, I find I am frequently having to toggle the responsive-mode on before I can check the layout on each device. I am always on the lookout for a shortcut to streamline my workflow. I'm toggling this on so many times that I frequently wish there was a way to have it on by default and then never have to touch the button again.

### Describe the solution you'd like

It would be nice if there was somewhere in the Elementor settings where you can choose to have the Responsive mode to be on or off be default.

**Comment by nicholaszein** [comment]
Hello there @mjagger88!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor **v3.12.0**! 🥳 

You can now use the new Editor Top Bar to have easy access to your most used commands without needing to have to toggle them on the bottom of the Editor Widget Panel.

https://elementor.com/help/top-bar-ui/

As this is still an experimental feature, we are still working on migrating old UIs, and some things might not be accessible right away, but for the most part, it works pretty well and should streamline your workflow. 😉 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
